proposals/init.md

```diff
@@ -75,6 +75,7 @@ An instance property containing an `init` accessor is considered settable in
 the following circumstances:
```

Comment:
@jcouv should this include Attribute property arguments? `[Attr(InitOnlyProp = true)]`